### DISCUSSION TITLE
Streaming payments contract

### DISCUSSION THREAD

**Discussion by rahimklaber** [stellar/sorobanathon#53] [General]
Hi, I created a contract for streaming payments. Check it out here https://github.com/rahimklaber/soroban_streaming_payments

@achallagundla

**Comment by achallagundla** [comment]
Heya @rahimklaber ,
Congratulations! Your contribution to Sorobanathon: First Light has been accepted! 🎉
Your submission may qualify for a reward of 1200 XLM! **One last step: to receive that reward, please fill out this [form]**(https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions. (You can also message me on discord @ anuxhya)

All the best,
Anuhya

**Reply by rahimklaber** [reply]
Hey @achallagundla, should I fill out the form If I already did it for another submission?
